### ISSUE TITLE
3.2.3 version is broken

### ISSUE THREAD

**Issue by yehor-manzhula** [A11yance/axobject-query#358]
Looks like 3.2.3 library version was published without rebuilding lib folder, so it contains source code for previous 3.2.2 version
that is still relies on deep-equal-json dependency that was deleted from package.json in the 3.2.3.
This causes the error:
```
Error: Cannot find module 'dequal/lite'
Require stack:
- /sample.nodejs/node_modules/axobject-query/lib/elementAXObjectMap.js
- /sample.nodejs/node_modules/axobject-query/lib/index.js
```

That is what I could see at the top in the lib/elementAXObjectMap.js file

```
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var _lite = require("dequal/lite");

// Rest of file contents
```

Could you please take a look on the issue?

**Comment by ljharb** [comment]
Whoops, thanks. I’ll have a fixed version published shortly.

**Comment by ljharb** [comment]
Fixed in v3.2.4; I added a prepack script so it can't be forgotten in the future. I also deprecated v3.2.3.

**Comment by yehor-manzhula** [comment]
@ljharb Confirmed, everything works as expected. Many thanks for the fix!
